youtube_dl/extractor/webofstories.py

```diff
@@ -98,3 +100,26 @@ def _real_extract(self, url):
             'description': description,
             'duration': duration,
         }
+
+
+class WebOfStoriesPlaylistIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?webofstories\.com/playAll/(?P<id>[^/]+)'
+    _TESTS = []
```

Comment:
I could definitely try, but can you please tell me how I can get only a single test to run. I don't want to run all other tests while I'm working on this one.